eventlet/green/ssl.py

```diff
@@ -29,19 +29,25 @@
 
 @contextmanager
 def _original_ssl_context(*args, **kwargs):
-    tmp_sslcontext = _original_wrap_socket.__globals__.get('SSLContext', None)
-    tmp_sslsocket = _original_sslsocket._create.__globals__.get('SSLSocket', None)
-    _original_sslsocket._create.__globals__['SSLSocket'] = _original_sslsocket
-    _original_wrap_socket.__globals__['SSLContext'] = _original_sslcontext
+    tmp_sslcontext = _original_wrap_socket.__globals__.get("SSLContext", None)
+    tmp_sslsocket = _original_sslsocket.sslsocket_class._create.__globals__.get(
+        "SSLSocket", None
+    )
+    _original_sslsocket.sslsocket_class._create.__globals__[
+        "SSLSocket"
+    ] = _original_sslsocket
+    _original_wrap_socket.__globals__["SSLContext"] = _original_sslcontext
     try:
         yield
     finally:
-        _original_wrap_socket.__globals__['SSLContext'] = tmp_sslcontext
-        _original_sslsocket._create.__globals__['SSLSocket'] = tmp_sslsocket
+        _original_wrap_socket.__globals__["SSLContext"] = tmp_sslcontext
+        _original_sslsocket.sslsocket_class._create.__globals__[
```

Comment:
please don't shrink it

there's max-line-width = 123 in tox.ini

if you want to introduce black formatter it's very much welcome in separate patch, with 123 width